sls invoke local failed with vanilla aws-groovy-gradle template

<!-- ⚠️⚠️ Acknowledge ALL below remarks -->
<!-- ⚠️⚠️ Request may not be processed if it doesn't meet outlined criteria -->

<!-- ⚠️⚠️ Ensure you're using *latest* version of a Framework -->
<!-- ⚠️⚠️ If you're uncertain you deal with a bug, ask first at https://forum.serverless.com -->
<!-- ⚠️⚠️ If your issue is influenced by a plugin, report at plugin repository, not here -->
<!-- ⚠️⚠️ Search existing issues to avoid creating duplicates  -->

<!-- ⚠️⚠️ Answer ALL the questions below -->

<!--
Q1: Describe the issue
-->

sls invoke local failed with vanilla aws-groovy-gradle template. You can reproduce the issue as below.

```
sls create -t aws-groovy-gradle
./gradlew build
sls invoke local -f hello
```

<!--
Q2: Provide (in below placeholder) FULL content of serverless.yml, ensuring that:
• It consistently reproduces described issue
• It's as minimal as possible
• There's no plugins involved (plugin related issues need to be reported at plugin repositories)
• Has sensitive parts masked out
-->

<details>
<summary><code>serverless.yml</code></summary>

I don't change the generated serverless.yml.

```yaml
# Welcome to Serverless!
#
# This file is the main config file for your service.
# It's very minimal at this point and uses default values.
# You can always add more config options for more control.
# We've included some commented out config examples here.
# Just uncomment any of them to get that config option.
#
# For full config options, check the docs:
#    docs.serverless.com
#
# Happy Coding!

service: t
# app and org for use with dashboard.serverless.com
#app: your-app-name
#org: your-org-name

# You can pin your service to only deploy with a specific Serverless version
# Check out our docs for more details
# frameworkVersion: "=X.X.X"

provider:
  name: aws
  runtime: java8

# you can overwrite defaults here
#  stage: dev
#  region: us-east-1

# you can add statements to the Lambda function's IAM Role here
#  iamRoleStatements:
#    - Effect: "Allow"
#      Action:
#        - "s3:ListBucket"
#      Resource: { "Fn::Join" : ["", ["arn:aws:s3:::", { "Ref" : "ServerlessDeploymentBucket" } ] ]  }
#    - Effect: "Allow"
#      Action:
#        - "s3:PutObject"
#      Resource:
#        Fn::Join:
#          - ""
#          - - "arn:aws:s3:::"
#            - "Ref" : "ServerlessDeploymentBucket"
#            - "/*"

# you can define service wide environment variables here
#  environment:
#    variable1: value1

# you can add packaging information here
package:
  artifact: ${file(build/build.json):distribution}

functions:
  hello:
    handler: com.serverless.Handler
#    The following are a few example events you can configure
#    NOTE: Please make sure to change your handler code to work with those events
#    Check the event documentation for details
#    events:
#      - http:
#          path: users/create
#          method: get
#      - websocket: $connect
#      - s3: ${env:BUCKET}
#      - schedule: rate(10 minutes)
#      - sns: greeter-topic
#      - stream: arn:aws:dynamodb:region:XXXXXX:table/foo/stream/1970-01-01T00:00:00.000
#      - alexaSkill: amzn1.ask.skill.xx-xx-xx-xx
#      - alexaSmartHome: amzn1.ask.skill.xx-xx-xx-xx
#      - iot:
#          sql: "SELECT * FROM 'some_topic'"
#      - cloudwatchEvent:
#          event:
#            source:
#              - "aws.ec2"
#            detail-type:
#              - "EC2 Instance State-change Notification"
#            detail:
#              state:
#                - pending
#      - cloudwatchLog: '/aws/lambda/hello'
#      - cognitoUserPool:
#          pool: MyUserPool
#          trigger: PreSignUp
#      - alb:
#          listenerArn: arn:aws:elasticloadbalancing:us-east-1:XXXXXX:listener/app/my-load-balancer/50dc6c495c0c9188/
#          priority: 1
#          conditions:
#            host: example.com
#            path: /hello

#    Define function environment variables here
#    environment:
#      variable2: value2

# you can add CloudFormation resource templates here
#resources:
#  Resources:
#    NewResource:
#      Type: AWS::S3::Bucket
#      Properties:
#        BucketName: my-new-bucket
#  Outputs:
#     NewOutput:
#       Description: "Description for the output"
#       Value: "Some output value"
```

</details>

<!--
Q3: Provide (in below placeholder) FULL name and output of command that exposes the problem.
       Note: Ensure SLS_DEBUG=* env var for verbose debug output
-->

<details>
<summary><b><code>sls invoke local -f hello</code> output</b></summary>

```
Serverless: In order to get human-readable output, please implement "toString()" method of your "ApiGatewayResponse" object.
log4j:WARN No appenders could be found for logger (com.serverless.Handler).
log4j:WARN Please initialize the log4j system properly.

log4j:WARN See http://logging.apache.org/log4j/1.2/faq.html#noconfig for more info.

java.lang.reflect.InvocationTargetException

	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.serverless.InvokeBridge.invoke(InvokeBridge.java:95)
	at com.serverless.InvokeBridge.<init>(InvokeBridge.java:39)
	at com.serverless.InvokeBridge.main(InvokeBridge.java:150)

Caused by: java.lang.RuntimeException: Unable to load FastStringService
	at org.apache.groovy.json.internal.FastStringUtils.getService(FastStringUtils.java:57)

	at org.apache.groovy.json.internal.FastStringUtils.toCharArray(FastStringUtils.java:67)
	at org.apache.groovy.json.internal.CharBuf.addJsonFieldName(CharBuf.java:524)
	at groovy.json.DefaultJsonGenerator.writeMapEntry(DefaultJsonGenerator.java:386)

	at groovy.json.DefaultJsonGenerator.writeMap(DefaultJsonGenerator.java:375)
	at groovy.json.DefaultJsonGenerator.writeObject(DefaultJsonGenerator.java:237)
	at groovy.json.DefaultJsonGenerator.writeObject(DefaultJsonGenerator.java:164)
	at groovy.json.DefaultJsonGenerator.toJson(DefaultJsonGenerator.java:98)

	at groovy.json.JsonOutput.toJson(JsonOutput.java:146)
	at com.serverless.Response.toJson(Response.groovy:14)
	at com.serverless.Handler.handleRequest(Handler.groovy:22)
	... 7 more
```

</details>

<!--
Q4: Provide (in below placeholder) output of serverless --version
-->

<b>Installed version</b>

```
Framework Core: 1.75.1
Plugin: 3.6.16
SDK: 2.3.1
Components: 2.32.0
```

aws-clojure-gradle failed too.

```
% sls create -t aws-clojure-gradle
[snip]
% ./gradlew build
[snip]
% sls invoke local -f hello
Serverless: In order to get human-readable output, please implement "toString()" method of your "ApiGatewayResponse" object.
Exception in thread "main" 
java.lang.ExceptionInInitializerError
	at clojure.lang.Namespace.<init>(Namespace.java:34)
	at clojure.lang.Namespace.findOrCreate(Namespace.java:176)
	at clojure.lang.Var.internPrivate(Var.java:153)
	at hello.<clinit>(Unknown Source)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at com.serverless.InvokeBridge.getInstance(InvokeBridge.java:69)
	at com.serverless.InvokeBridge.<init>(InvokeBridge.java:37)
	at com.serverless.InvokeBridge.main(InvokeBridge.java:150)
Caused by: java.io.FileNotFoundException: Could not locate clojure/core__init.class or clojure/core.clj on classpath.
	at clojure.lang.RT.load(RT.java:463)
	at clojure.lang.RT.load(RT.java:426)
	at clojure.lang.RT.doInit(RT.java:468)
	at clojure.lang.RT.<clinit>(RT.java:336)

	... 9 more
```

--- a/lib/plugins/aws/invokeLocal/java/src/main/java/com/serverless/InvokeBridge.java
+++ b/lib/plugins/aws/invokeLocal/java/src/main/java/com/serverless/InvokeBridge.java
@@ -65,6 +65,7 @@ public class InvokeBridge {
   private Object getInstance() throws Exception {
     URL[] urls = {this.artifact.toURI().toURL()};
     URLClassLoader child = new URLClassLoader(urls, this.getClass().getClassLoader());
+    Thread.currentThread().setContextClassLoader(child);
 
     this.clazz = Class.forName(this.className, true, child);
 

